spec/models/user_spec.rb

```diff
@@ -48,4 +48,11 @@
     user.email = "asdlkfjalgalwrg"
     expect(user).to be_invalid
   end
+
+  it "must have a reference an order" do
+    user.orders.build(total: 200, user_id: user.id, status: "completed")
+    expect(user.orders.map(&:total)).to eq([200])
+    expect(user.orders.map(&:status)).to eq(['completed'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.